cpp/tests/search/search_list_test.cpp

```diff
@@ -347,3 +347,66 @@ TYPED_TEST(TypedListContainsTestColumnNeedles, ListsOfStructs)
   auto const result   = cudf::contains(*haystack, *needles);
   CUDF_TEST_EXPECT_COLUMNS_EQUAL(expected, *result, verbosity);
 }
+
+struct ListLowerBound : public cudf::test::BaseFixture {
+};
+
+TEST_F(ListLowerBound, ListWithNulls)
+{
+  {
+    using lcw = cudf::test::lists_column_wrapper<double>;
+    using cudf::test::iterators::nulls_at;
```

Comment:
This is already imported at https://github.com/rapidsai/cudf/blob/681039ae9e662dd160b91d5db083515357f0f583/cpp/tests/search/search_list_test.cpp#L28.
```suggestion
```

Reply:
I think it's cleaner to use only specific names rather than the whole namespace, but I wouldn't invest a lot of time on this.

Reply:
I will update the existing line to only promote specific names instead of the whole namespace